### DISCUSSION TITLE
if webpackBundler, how to dispose proxy?

### DISCUSSION THREAD

**Discussion by miss7cd** [vuepress/core#1027] [Q&A] [answered]
Am  I missing an Reference or Suggestions in doc? Thanks for your help.

**Comment by Mister-Hope** [comment, marked as answer]
https://v2.vuepress.vuejs.org/reference/bundler/webpack.html#configurewebpack

https://v2.vuepress.vuejs.org/reference/bundler/webpack.html#chainwebpack

**Reply by miss7cd** [reply]
Thanks for your help
```
chainWebpack: (config, isServer, isBuild) => {
            config.devServer.host('localhost').port('3010').proxy({
                '/api': {
                    target:  ...,
                    changeOrigin: true
                }
            })
        }
```
but does not work, and do this, the proxy is work

![vuepress](https://user-images.githubusercontent.com/38612985/184521388-1917f84a-cd23-4e45-afc3-17258bd9c3a1.png)

that's I missing something or a bug? 
Thanks !
